style.css

```diff
@@ -50,23 +50,55 @@ li a:hover:not(.active) {
   background-color: #04AA6D;
 }
 
-img#lens {
-  max-width: 250px;
-  margin: 0 auto;
+figure {
+  display: table;
+  flex-flow: column;
+  margin: 2rem;
+  padding: 5px;
+}
+
+figure.centered {
+  margin: 2rem auto;
+  clear: both;
+}
+
+figure img {
   display: block;
+  padding-bottom: 5px;
+}
+
+figcaption {
+  caption-side: bottom;
+  display: table-caption;
+  font-style: italic;
+  font-size: smaller;
+  padding: 0.5em;
+  text-align: center;
+}
+
+.floatRight {
+  clear: left;
+  float: right;
+  margin: 0 0 10px 10px;
 }
 
 img#irises {
-  width: 100%;
-  max-width: 1000px;
   margin: 0 auto;
-  display: block;
+  width: 100%;
 }
 
 img {
+  display: inline-block;
+  object-fit: contain;
   width: 100%;
   height: 100%;
-  object-fit: contain;
+  max-width: max-content;
```

Comment:
These max settings cause the images to never get larger than they actually are when the page is resized.